### DISCUSSION TITLE
The theme

### DISCUSSION THREAD

**Discussion by sirhaffy** [asmvik/yabai#1215] [General]
This is off-topic, but I love the theme in the screenshot. 
Please what theme and font is it? Would love to download it.

**Comment by asmvik** [comment]
The font is Fantasque Sans Mono: https://github.com/belluzj/fantasque-sans

The theme is just a set of colours that I put together. 
I don't have them available as a "proper" theme, it is just configured in my terminal and neovim config(s) to work on my machine.

Neovim theme: https://github.com/koekeishiya/dotfiles/blob/master/nvim/colors/koe.vim
Neovim Airline plugin theme: https://github.com/koekeishiya/dotfiles/blob/master/nvim/autoload/airline/themes/koe.vim
Tmux: https://github.com/koekeishiya/dotfiles/blob/master/tmux/tmux.conf
Kitty (terminal): https://github.com/koekeishiya/dotfiles/blob/master/kitty/kitty.conf

**Reply by sirhaffy** [reply]
Thanks! To bad it was not a finished VS Code theme, many I will try to make one out of these colors. Very much lime the combo.

**Reply by Shpackk** [reply]
That's sad. Wasted 10-20 min to find this theme lol
